modules/docs/mdx/changelog.stories.mdx

```diff
@@ -3,4 +3,5 @@ import Changelog from '../../../CHANGELOG.md';
 import Markdown from '../../../utils/storybook/Markdown.tsx';
 
 <Meta title="Welcome/Changelog" />
+
```

Comment:
The changelog wasn't working before... This is annoying. Maybe a prettier update can fix this